### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by KittNeKit**

### CODE REVIEW

File: taxi/views.py [hunk 1]
```diff
@@ -57,7 +83,27 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
...
+
+        context["search_form"] = CarModelSearchForm(
+            initial={
+                "model": model
+            }
+        )
+        return context
+
+    def get_queryset(self):
+        queryset = Car.objects.all().select_related("manufacturer")
```
Comment (Bohdan-Salamakha) [inline taxi/views.py:100]: remove redundant .all()

File: taxi/views.py [hunk 2]
```diff
@@ -111,7 +178,7 @@ class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
     if (
-        Car.objects.get(id=pk) in driver.cars.all()
+            Car.objects.get(id=pk) in driver.cars.all()
```
Comment (Bohdan-Salamakha) [inline taxi/views.py:181]: use get_object_or_404() to avoid Error

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]

File: taxi/views.py (hunk 1 above)
Comment (Bohdan-Salamakha) [reply taxi/views.py:100]: not fixed

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]

File: templates/taxi/car_list.html [hunk 3]
```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+      <form action="" method="get" class="form-inline">
+      {{ search_form|crispy }}
+      <input type="submit" value="🔍">
+    </form>
```
Comment (olena-hul) [inline templates/taxi/car_list.html:14]: Move it to a template and include where needed

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]

**Pull request closed without merging**